app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -104,15 +104,19 @@
 
         .notes-view {
           &__gloss {
-            max-width: 10rem;
+            .item-maori-name,
+            .item-name {
+              padding: .3rem .2rem .3rem .2rem;
```

Comment:
🐘 This can be simplified to `padding: .3rem .2rem;`

Reply:
Thank you!